Modules/Devices/src/Devices.Application/Tiers/Commands/DeleteTier/DeleteTierCommand.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Backbone.Modules.Devices.Domain.Aggregates.Tier;
+using MediatR;
+
+namespace Backbone.Modules.Devices.Application.Tiers.Commands.DeleteTier;
+
+public class DeleteTierCommand : IRequest<DeleteTierResponse>
```

Comment:
```suggestion
public class DeleteTierCommand : IRequest
```
(don't forget to delete the `DeleteTierResponse`